ntbs-service/Helpers/ExtensionMethods.cs

```diff
@@ -14,5 +17,19 @@ public static class DateTimeExtension
         }
     }
 
+    public static class NotificationExtensions

```

Comment:
`NotificationsExtensions` (plural)? Not sure what the convention is :)

Reply:
I thought I'd keep it more general, in case we want to define any other extensions on notification (not just on enumerables)